connection.js

```diff
@@ -763,13 +758,11 @@ function genericAllowedFamiliesReject(argNameString, notificationUnit) {
     } else if (argFamilySingularString.includes("komi") && (notificationUnit === null)) {
         notificationUnitConverted = "automatic";
     }
-    conn_log(`${argFamilySingularString} ${rankedUnranked} `
-             + `-${notificationUnitConverted}-, not in -${argValueString}- `);
-    const msg = `${argFamilySingularString} -${notificationUnitConverted}- `
-                + `is not allowed on this bot ${rankedUnranked}, please `
-                + `choose one of these allowed ${argFamilySingularString}s `
-                + `${rankedUnranked}: -${argValueString}-`;
-    return { reject: true, msg };
+    const reason = `${argFamilySingularString} -${notificationUnitConverted}- `
+                   + `is not allowed on this bot ${rankedUnranked}`;
+    const demand = ` , please choose one of these allowed ${argFamilySingularString}s `
```

Comment:
Another redundant space at start of `demand`.

Reply:
indeed, same as https://github.com/online-go/gtp2ogs/pull/286#discussion_r426189771